apps/functions-subscription/src/adapters/azure/cosmosdb/activation-request.ts

```diff
@@ -67,6 +67,31 @@ export const makeActivationRequestReaderWriter = (
         ),
         TE.flatMapEither(decodeFromFeed(ActivationRequestCodec)),
       ),
+    get: (trialId, userId) =>
+      pipe(
+        TE.tryCatch(
+          () =>
+            container.items
+              .query({
+                query:
+                  'SELECT * FROM c WHERE c.trialId = @trialId AND c.type = "request" AND c.userId = @userId OFFSET 0 LIMIT 1',
```

Comment:
**note**: we have to check the performances and the RU consumed by this query, because there isn't an index with `userId` property

Reply:
moreover, check if there is a better way to fetch only one document. by design, it cannot be possible to have multiple documents with same `trialId` and `userId`

Reply:
Index created with commit [4ea798d](https://github.com/pagopa/trial-system/pull/160/commits/4ea798d412f1e7b1c85e23cb934baceda9d8a72f)